### PR TITLE
Update calculation of light and historic demand in `westeros_baseline` tutorial

### LINKED ISSUE #522: `westeros_baseline` tutorial should use `demand_per_year`
In the [`westeros_baseline`](https://github.com/iiasa/message_ix/blob/main/tutorial/westeros/westeros_baseline.ipynb) tutorial the `light_demand` should be calculated out of the `demand_per_year` and the `gdp_profile`. Right now only the `gdp_profile` is multiplied by 100 which is then be used as the demand.

### PR DESCRIPTION
**Pull request by LauWien**
This PR updates the calculation of the `light_demand` and `historic_demand` in the `westeros_baseline` tutorial. Closes #522 

## How to review

- Read the diff and note that the CI checks all pass.

## PR checklist

- [x] Continuous integration checks all ✅
- [x] Update release notes.

### CODE REVIEW

**Comment by LauWien** [comment]
> 
> 
> Thanks @LauWien for opening this and resolving inconsistencies. As for the MACRO, I'd suggest you update the Excel file and make the tests pass. Because the layout of data in that Excel file is being used by different modeling streams (global, national, tutorials) for doing MACRO calibration. There are also some suggestions for improving the Excel layout in PR #327. Later, we can decide what to do with that Excel file.

Thank you for your review! I'll do as suggested :)

File: message_ix/tests/test_macro.py [hunk 1]
```diff
@@ -208,10 +211,23 @@ def test_calibrate_roundtrip(westeros_solved):
     # this is a regression test with values observed on Aug 9, 2019
     with_macro = westeros_solved.add_macro(W_DATA_PATH, check_convergence=True)
     aeei = with_macro.par("aeei")["value"].values
-    npt.assert_allclose(aeei, 1e-3 * np.array([20, 71.759022, 37.424904, 19.936694]))
+    npt.assert_allclose(
+        aeei,
+        1e-3
+        * np.array([20, -7.514287920171298, 43.62562814596503, 21.143463116337838]),
```
Comment (khaeru) [inline message_ix/tests/test_macro.py:217]: The default relative tolerance for [`assert_allclose`](https://numpy.org/doc/stable/reference/generated/numpy.testing.assert_allclose.html) is 1e-7, so we don't need to include more than that many significant figures.

File: message_ix/tests/test_macro.py [hunk 2]
```diff
@@ -208,10 +211,23 @@ def test_calibrate_roundtrip(westeros_solved):
...
+    )
     grow = with_macro.par("grow")["value"].values
     npt.assert_allclose(
-        grow, 1e-3 * np.array([26.583631, 69.101286, 79.520269, 24.529274])
+        grow,
+        1e-3
+        * np.array(
+            [
+                26.583631304232025,
```
Comment (khaeru) [inline message_ix/tests/test_macro.py:225]: Ditto.

File: message_ix/testing/__init__.py [hunk 3]
```diff
@@ -507,9 +506,8 @@ def make_westeros(mp, emissions=False, solve=False, quiet=True):
             make_df(name, **common, technology=tec, commodity=c, level=l, value=value),
         )
 
-    # FIXME the value for wind_ppl should be 0.36; requires adjusting other tests.
```
Comment (khaeru) [inline message_ix/testing/__init__.py:510]: Thanks for going through and making those adjustments! 🙏🏾

**Review by khaeru — CHANGES_REQUESTED** [review]
Looks great, thanks for this clean-up. Only a few minor items to tidy.

**Comment by LauWien** [comment]
> 
> 
> Looks great, thanks for this clean-up. Only a few minor items to tidy.

Thank you for your review :)

**Review by khaeru — APPROVED** [review]
There are two more places where we could have also reduced the numbers of figures: test_macro.py around line 138, and test_reporting.py. 

However, LGTM! Those can be tidied the next time a test change requires adjustment to those values.

**Comment by LauWien** [comment]
> 
> 
> There are two more places where we could have also reduced the numbers of figures: test_macro.py around line 138, and test_reporting.py.
> 
> However, LGTM! Those can be tidied the next time a test change requires adjustment to those values.

Oh yes, I'll do it in this PR https://github.com/iiasa/message_ix/pull/524, where I need to update them again, anyways :) Was maybe not the best idea to open two PRs with changing the same tutorial.

**Pull request merged**